File: devops-scm-api/src/main/java/com/tencent/devops/scm/api/pojo/Oauth2AccessToken.java
package com.tencent.devops.scm.api.pojo;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class Oauth2AccessToken {

    private String accessToken;
    private String tokenType;
    // 单位: 秒
    private Long expiresIn;
    private String refreshToken;
    private String scope;
}

File: devops-scm-api/src/main/kotlin/com/tencent/devops/scm/api/pojo/Oauth2AccessToken.kt
package com.tencent.devops.scm.api.pojo

data class Oauth2AccessToken(
    // 访问令牌
    val accessToken: String,
    // 令牌类型
    val tokenType: String,
    // 过期时间(单位:秒)
    val expiresIn: Long,
    // 刷新令牌
    val refreshToken: String,
    // 权限范围
    val scope: String? = null
)
